core-feature-pack/common/src/main/resources/content/bin/standalone.sh

```diff
@@ -343,7 +347,6 @@ while true; do
          -mp \""${JBOSS_MODULEPATH}"\" \
          org.jboss.as.standalone \
          -Djboss.home.dir=\""$JBOSS_HOME"\" \
-         -Djboss.server.base.dir=\""$JBOSS_BASE_DIR"\" \
```

Comment:
Hi @yersan, yes I agree, I didn't notice that before. So what if we keep checking, if `SERVER_OPTS` contains `-Djboss.server.base.dir`, if so, we will use `SERVER_OPTS` as they are, however if not `JBOSS_BASE_DIR` will be added to `SERVER_OPTS` and that way, the directory configured on `JAVA_OPTS` could be eventually passed.

Reply:
Hi @lvydra , I think we should prefer the one coming from `JBOSS_BASE_DIR` instead of the one added at `SERVER_OPTS`.

The `JBOSS_BASE_DIR` is a consolidated one and looking at the code we have explicitly added computation to consolidate it, e.g symlinks won't be there and it will use the absolute path.

It is a bit weird because having both as server arguments we will get the latest one but that seems unintentional.

About `JAVA_OPTS`, it is harmless if it is defined there since the server won't use it. We could remove it from that variable after consolidating the setting on `JBOSS_BASE_DIR`, but I think we could keep it there since the argument is the server doesn't use it and we don't want to compromise any possible user application that is dealing with the value by using that property. After all, is a user-defined variable and shown as part of the launch, it will be even weirder to not see it there if you have defined it.